ti/instrumentation/java/bazel.go

```diff
@@ -57,21 +99,28 @@ func (b *bazelRunner) GetCmd(ctx context.Context, tests []ti.RunnableTest, userA
 	// Use only unique classes
 	pkgs := []string{}
 	clss := []string{}
-	set := make(map[string]interface{})
 	ut := []string{}
+	rls := []string{}
 	for _, t := range tests {
-		if _, ok := set[t.Class]; ok {
-			// The class has already been added
-			continue
-		}
-		set[t.Class] = struct{}{}
 		ut = append(ut, t.Class) //nolint:staticcheck
 		pkgs = append(pkgs, t.Pkg)
 		clss = append(clss, t.Class)
+		rls = append(rls, t.Autodetect.Rule)
 	}
 	rulesM := make(map[string]struct{})
 	rules := []string{} // List of unique bazel rules to be executed
+	set := make(map[string]interface{})
```

Comment:
Please use variable names such that they indicate what they hold. set isn't readable.

Reply:
Done